pkg/commands/deploy.go

```diff
@@ -48,10 +48,10 @@ func NewRemoveCommand(commonOpts *CommonOptions) *cobra.Command {
 		Use:   "remove",
 		Short: "remove the components and configurations needed for topology-aware-scheduling",
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
in this case copying the option is intended, it emphasizes the fact the function will NOT mutate the argument.
Furthermore, in this case the copies are not really performance sensitive, so overall the added bit of clarity justifies the extra copy.

Reply:
Thank you for the clarification